core/components/molecules/pagination/pagination-toolbar.md

```diff
@@ -0,0 +1,71 @@
+```meta
```

Comment:
I don't think we need to document `pagination-toolbar`, it's an internal component 

(p.s. it also breaks on the docs right now)

Reply:
Yep I know. Working on that, so should we completely remove that component from docs?

Reply:
> I don't think we need to document pagination-toolbar, it's an internal component

We need to show how to use this component somehow. The easiest way, for now, is to move it to our components list (not inside pagination, but as a standalone component).

Reply:
Ah, you're right.

I assumed wrongly that `pagination-toolbar` is a child of `pagination`, it's a separate compound component